src/CustomProviders/examples/New-AzCustomProvider.md

```diff
@@ -0,0 +1,24 @@
+### Example 1: Create a custom provider
+```powershell
+PS C:\> New-AzCustomProvider -ResourceGroupName myRG -Name Namespace.Type -Location "West US 2" -ResourceType @{Name="CustomRoute1"; Endpoint="https://www.contoso.com/"}
+
+
+Location  Name             Type
+--------  ----             ----
+West US 2 Namespace.Type   Microsoft.CustomProviders/resourceproviders
+```
+
+Create a custom resource provider
+
+### Example 2: Create a custom provider with associations
+```powershell
+PS C:\> New-AzCustomProvider -ResourceGroupName myRG -Name Namespace.Type -Location "West US 2" -ResourceType @{Name="CustomRoute1"; Endpoint="https://www.contoso.com/"}, @{Name="Associations"; Endpoint="https://contoso.com/myService", RoutingType="Proxy,Cache,Extension"}
+
+Location  Name             Type
+--------  ----             ----
+West US 2 Namespace2.Type   Microsoft.CustomProviders/resourceproviders
```

Comment:
It looks like the name returned is non consistent with the the command

Reply:
Eagle eye. Fixed